### DISCUSSION TITLE
An error occurred when importing the beatmap edited by lazer into stable

### DISCUSSION THREAD

**Discussion by Rain-0x01-39** [ppy/osu#28747] [Q&A] [answered]
![error window from stable](https://github.com/ppy/osu/assets/83620631/fe84c26a-2091-4fba-892d-2c57cf204ba7 "error window from stable")

**Comment by cdwcgt** [comment, marked as answer]
https://github.com/ppy/osu/issues/28607
has been fixed in next release
